src/components/DesktopHead/DesktopHead.jsx

```diff
@@ -17,7 +17,6 @@ export const DesktopHead = props => {
     menuExpand,
     setShowMapControls,
     showMapControls,
-    pageExpand,
```

Comment:
this was just a redundant state, we could achieve the same thing with only having the `pageShown`